README.md

```diff
@@ -120,7 +120,7 @@ Licenses of minimal dependencies:
 | required package | License            	|
 |------------------|------------------------|
 | gevent     	   | MIT                	|
-| flask      	   | BSD 3-clause       	|
+| klein     	   | MIT      	            |
```

Comment:
This is an issue with github diffs' preview: if you look at the raw file, it must be aligned. 😄